spyder/app/mainwindow.py

```diff
@@ -274,10 +274,7 @@ def qt_message_handler(msg_type, msg_log_context, msg_string):
 # =============================================================================
 # Dependencies
 # =============================================================================

```

Comment:
Please remove these three lines above this comment because they are not needed anymore.